extensions/roc-plugin-dredd/src/roc/index.js

```diff
@@ -1,7 +1,32 @@
+import { lazyFunctionRequire } from 'roc';
+
 import config from '../config/roc.config.js';
 import meta from '../config/roc.config.meta.js';
 
+const lazyRequire = lazyFunctionRequire(require);
+
 export default {
     config,
-    meta
+    meta,
+    actions: [
+        {
+            hook: 'server-started',
+            description: 'Run dredd tests',
+            action: lazyRequire('../actions/dredd'),
```

Comment:
With the current setup this action would start at all times when the `server-started` hook is invoked and this might not be desired.

One alternative is to register the action when the `dredd` command is invoked, this would mean that it does not exist when not running that command. As a result it would not be invoked any other times than when `dredd` is used with the downside that Roc will not be able to generate documentation for it. 

An alternative is make the action itself conditional based on the context for example.